docs/HOW_TO_USE.md

```diff
@@ -23,12 +23,8 @@ locations (in this order):
 On POSIX systems:
 ```
 <EXE-DIR>/nzbget.conf
-~/.nzbget
```

Comment:
Apparently this path was accidentally removed here.

Reply:
It wasn't accidental; `~/.nzbget` will be considered deprecated from now on.